.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: ci 
+on: push
```

Comment:
This may need to get removed since we may run out of quota in the future if we merge alot of changes
https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions#about-billing-for-github-actions